keyboards/s7_elephant/keymaps/default/keymap.c

```diff
@@ -0,0 +1,95 @@
+ * `------'   `-------------------------------------------------------------------------------------------------'
+ */
+
+[_QWERTY] = LAYOUT_ansi(
+KC_F1, KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_INS,
+KC_F2, KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,
+KC_F3, KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,      KC_ENT,      KC_HOME,
+KC_F4, KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
+KC_F5, KC_LCTL, KC_LGUI, KC_LALT,                      KC_SPC,                          MO(_FN), KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
+),
+
+/* FN
+ * ,------.   ,-------------------------------------------------------------------------------------------------.
+ * |Reset |   |  F1   | F2  | F3  | F4  | F5  | F6  | F7  | F8  | F9  | F10 | F11 | F12 | PScr |  ScrLck  |Pause|
+ * |------|   |-------------------------------------------------------------------------------------------------|
+ * |      |   |        |     |     |     |     |     |     |     |     |     |     |     |     |          |     |
+ * |------|   |-------------------------------------------------------------------------------------------------|
+ * |      |   |         |Prev |Play |Next |     |     |     |     |VolD |Mute |VolU |     |               |PgUp |
+ * |------|   |-------------------------------------------------------------------------------------------------|
+ * |      |   |           |     |     |     |     |     |     |     |     |     |     |             |     |PgDn |
+ * |------|   |-------------------------------------------------------------------------------------------------|
+ * |EPRM  |   |      |       |       |                                   |         |           |     |    |     |
+ * `------'   `-------------------------------------------------------------------------------------------------'
+ */
+
+[_FN] = LAYOUT_ansi(
+RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_SLCK, KC_PAUS,
+_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+_______, _______, KC_MPRV, KC_MPLY, KC_MNXT, _______, _______, _______, _______, KC_VOLD, KC_MUTE, KC_VOLU, _______, _______,          KC_PGUP,
+_______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PGDN,
+EPRM,    _______, _______, _______,                        _______,                       _______, _______,          _______, _______, _______
```

Comment:
You should use the official keycode for this: 
```suggestion
EEP_RST, _______, _______, _______,                        _______,                       _______, _______,          _______, _______, _______
```

Reply:
Fixed.